--- common.py.orig	2007-07-18 23:30:33.000000000 +0300
+++ common.py	2007-07-18 23:32:34.000000000 +0300
@@ -16,6 +16,9 @@
 # program path, set in wxglade.py
 wxglade_path = '.'
 
+# documentation path
+wxglade_docs_path = '/usr/share/doc/wxGlade-0.5'
+
 # widgets dictionary: each key is the name of some EditWidget class; the mapped
 # value is a 'factory' function which actually builds the object. Each of these
 # functions accept 3 parameters: the parent of the widget, the sizer by which
--- main.py.orig	2007-07-18 23:00:16.000000000 +0300
+++ main.py	2007-07-18 23:30:57.000000000 +0300
@@ -780,7 +780,7 @@
         self.about_box.ShowModal()
 
     def show_tutorial(self, event):
-        docs_path = os.path.join(common.wxglade_path, 'docs', 'index.html')
+        docs_path = os.path.join(common.wxglade_docs_path, 'docs', 'index.html')
         if wx.Platform == "__WXMAC__":
             os.system('open -a Help\ Viewer.app %s' % docs_path)
         else:
